cmake/ImportLLVM.cmake

```diff
@@ -16,7 +16,9 @@
 
 #[=======================================================================[.rst:
 Module imports LLVM into the ComputeAorta build by pulling in LLVM CMake
-modules from the ``install`` directory of a build.
+modules from the ``install`` directory of a build. This only happens of we have not
```

Comment:
I think there's also an argument for taking `CA_IN_LLVM_TREE` out of `ImportLLVM`, and only calling into `ImportLLVM` if `CA_IN_LLVM_TREE`/`OCK_IN_LLVM_TREE` is false. That way the logic of `ImportLLVM` is clearer - it imports LLVM. If we're already inside LLVM, we don't need to do that.

See how the documentation still says `User option CA_LLVM_INSTALL_DIR is required to be set to the filepath of the LLVM install before including this module`. This documentation is now incorrect and/or misleading.

I think if this module is to auto-detect being in-tree, it should probably be renamed (to `DetectLLVM` or something) and redocumented.

Reply:
In fact, none of the documentation after this paragraph is correct for `CA_IN_LLVM_TREE`.

Reply:
Also this new documentation overflows the 80 character column limit.